css/loading.css

```diff
@@ -0,0 +1,62 @@
+#graphiql-container .spinner-container {
+  position: absolute;
+  top: 20%;
+  height: 36px;
+  width: 36px;
+  left: 50%;
+  transform: translate(-50%,-50%);
+  z-index: 10;
+}
+
+#graphiql-container .spinner {
+  vertical-align: middle;
+  display: inline-block;
+  height: 24px;
+  width: 24px;
+  position: absolute;
+  -webkit-animation: rotation .6s infinite linear;
+  -moz-animation: rotation .6s infinite linear;
+  -o-animation: rotation .6s infinite linear;
+  animation: rotation .6s infinite linear;
+  border-left: 6px solid rgba(150, 150, 150, .15);
+  border-right: 6px solid rgba(150, 150, 150, .15);
+  border-bottom: 6px solid rgba(150, 150, 150, .15);
+  border-top: 6px solid rgba(150, 150, 150, .8);
+  border-radius: 100%;
+}
+
+@-webkit-keyframes rotation {
+  from {
+  -webkit-transform: rotate(0deg);
```

Comment:
nit: indent inside the {} or one-line it. I think it may be nicer to read:

```css
from { -crap-transform: rotate(0deg); }
to { -crap-transform: rotate(359deg); }
```